src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -70,7 +71,12 @@
     private static final long EXPIRES_NEVER = Long.MAX_VALUE;
     private static final long EXPIRES_IMMEDIATELY = 0L;
 
-    private static final int TOKEN_EXPIRED_SIGNAL = 1;
+    private static final int CHALLENGE_MASK = 0x01;
+    private static final int CHALLENGE_BIT = 1;
+
+    private static final int GRANT_VALIDATION_SIGNAL = 1;
+
+    private static final String CHALLENGE_AFTER_CLAIM = "caf";
```

Comment:
Thinking about the complexity of setup, it is probably easier for the token generator to always include the same claim with relative challenge delta value that we can then make absolute by comparing to expiration, rather than requiring token generator to provide an absolute challenge-after timestamp claim.

Reply:
Let's move this name to `OAuthConfiguration` defaulting to `https://reaktivity.org/challenge_response_timeout`.
Actually, let's separate the namespace and name constants in `OAuthConfiguration`, then recombine them in the factory.